CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ Mapbox welcomes participation and contributions from everyone.
 - Improved behavior of enhanced location teleports on parallel roads, especially forks. [#5768](https://github.com/mapbox/mapbox-navigation-android/pull/5768)
 - Changed the internal vanishing route line feature implementation to use the `line-trim-offset` property of `LineLayer` to improve the performance of updates. The optimization is not available when `MapboxRouteLineOptions#styleInactiveRouteLegsIndependently` is enabled. [#5697](https://github.com/mapbox/mapbox-navigation-android/pull/5697)
 
+#### Known issues
+- `RestStop#name` are only available through `RoadObjectsOnRouteObserver` and not through `EHorizonObserver` yet.
```

Comment:
Did you use the plural form intentionally here?
```suggestion
- `RestStop#name` is only available through `RoadObjectsOnRouteObserver` and not through `EHorizonObserver` yet.
```

Reply:
Intentionally, there can be multiple rest stops along the route, hence the plural form :shrug: